public/posts/chapter-116.md

```diff
@@ -2,7 +2,116 @@
+3. **Stealing Information**: When you enter your personal details on the fake website, the hacker steals them.
+
+### Types of Phishing:
+
+1. **Email Phishing**:
+   - Most common type of phishing where fake emails ask you to provide personal information or download harmful attachments.
+   - **Example**: An email that looks like it’s from your bank, asking you to “verify” your account details by clicking a link.
+
+2. **Spear Phishing**:
+   - More personalized phishing attacks. The hacker uses specific information about you (like your name or job) to make the email seem more believable.
+   - **Example**: An email from what looks like your boss asking for sensitive work information.
+
+3. **Whaling**:
+   - Aimed at high-profile targets like company executives or government officials.
+   - **Example**: An executive receives an email that seems to come from the company’s legal team, asking for confidential documents.
+
+4. **Vishing**:
+   - Phishing over the phone, where attackers pretend to be a trusted source and ask for sensitive information.
+   - **Example**: A scammer calls, claiming to be from your bank, and asks for your account details to "fix" an issue.
+
+5. **Smishing**:
+   - Phishing through text messages (SMS). The message might include a link or a request for personal information.
+   - **Example**: A text message that looks like it's from a delivery service asking you to confirm your address by clicking a link.
+
+---
+
+## How to Protect Yourself from Cyber Threats
+
+### 1. Keep Your Software Up to Date
+- Always install the latest updates for your operating system, browser, and apps. These updates often fix security holes that hackers might exploit.
+
+### 2. Use Antivirus Software
+- Antivirus programs can detect and remove viruses and malware. Make sure it's always up to date.
+
+### 3. Be Careful with Emails and Links
+- Don’t open attachments or click on links in emails from unknown senders. Even if the email looks legit, it’s safer to visit the company’s website directly.
+
+### 4. Use Strong Passwords
+- Create strong, unique passwords for each of your accounts. Avoid using the same password for multiple websites. You can use a password manager to help you keep track.
+
+### 5. Enable Two-Factor Authentication (2FA)
+- Two-factor authentication adds an extra layer of security by requiring you to enter a code from your phone along with your password.
+
+### 6. Back Up Your Data
+- Regularly back up your files to an external hard drive or cloud service. If you get infected by ransomware, you won’t lose everything.
+
+---
+
+## Conclusion
+
+Cyber threats like viruses, malware, and phishing are constantly evolving, but by understanding these threats and taking precautions, you can significantly reduce the risk of being targeted. Stay vigilant, keep your software up to date, and always be cautious when sharing personal information online.
+
+---
 Submit a PR at https://github.com/org-45/computer-book-for-kids
```

Comment:
Thanks for the contribution. You can remove the last line as we already have the content now. There are few punctuation related errors but thats ok for now.